### PR TITLE
Khwang/add users to transactions

### PR DESCRIPTION
**Pull request by khwang**
1. Ensures that all transactions are assigned to a given user.
2. Updates the transaction view so that you only see transactions assigned to yourself.
3. Updates seed data with a default test user and some test data.

Note: You will need to `rake db:reset` to get these changes!

### CODE REVIEW

File: db/migrate/20160513023243_create_transactions.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ def change
       t.string :title
       t.date :date
       t.integer :amount
+      t.references :user, index: true, foreign_key: true, null: false
```
Comment (Radahika) [inline db/migrate/20160513023243_create_transactions.rb]: remove space.

File: db/seeds.rb [hunk 2]
```diff
@@ -6,6 +6,8 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-Transaction.create(title: "Transaction 1", date: Date.parse('2016-05-01'), amount: 100)
-Transaction.create(title: "Transaction 2", date: Date.parse('2016-05-02'), amount: 299)
-Transaction.create(title: "Transaction 3", date: Date.parse('2016-05-03'), amount: 50)
+user = User.create! email: "test@test.com", password: "asdfasdf", password_confirmation: "asdfasdf"
+
```
Comment (Radahika) [inline db/seeds.rb]: can we get multiple users to make sure user data is independent?

**Comment by Radahika** [comment]
I know this annoying to set up, but it would be nice if we had a simple test showing how to create a user and that you can add/delete transactions.

File: db/migrate/20160513023243_create_transactions.rb (hunk 1 above)
Comment (khwang) [inline db/migrate/20160513023243_create_transactions.rb]: Done

File: db/seeds.rb (hunk 2 above)
Comment (khwang) [inline db/seeds.rb]: Done

**Comment by khwang** [comment]
Updated the tests and fixed them.

**Comment by Radahika** [comment]
LGTM

**Pull request merged**